### DISCUSSION TITLE
Come monitorare un telefono Android?

### DISCUSSION THREAD

**Discussion by stephaneklm** [indomus/forum#152] [Hardware & Firmware]
Prima di tutto ciao a tutti.

Ti invito a trovare idee per risolvere il mio problema.
Voglio sapere c'è un modo per monitorare i telefoni Android, come “Trova il mio iPhone”,, questo dispositivo consente agli utenti di rintracciare il proprio telefono quando il telefono viene rubato o smarrito. 

Se qualcuno mi fa venire in mente un'idea che mi interessa (se mi supporto, posterò una soluzione, chissà che potrebbe aiutare qualcuno)

Grazie mille!

**Comment by snakuzzo** [comment]
Puoi farlo semplicememte installando l'app ufficiale sullo smartphone e collegandoti alla tua istanza di HA. Tra le tante cose verrà creata un'enttità di tipo `device_tracker` con la quale potrai monitorare (anche su mappa) la posizione del dispositivo.

**Comment by stephaneklm** [comment]
Grazie mille per la tua risposta, e allo stesso tempo mi dispiace molto, perché ho installato Android Device Manager sotto l'introduzione di [questo articolo](https://pctempo.com), che ha perfettamente risolto il mio problema. Voglio mantenere questo argomento in modo da poter aiutare più persone. Grazie ancora!

**Reply by snakuzzo** [reply]
È vietato postare link a siti terzi
